### PR TITLE
Add Accordion docs

### PR DESCRIPTION
**Pull request by alimpens**

### CODE REVIEW

File: packages/css/src/accordion/README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
 # Accordion
 
-Het accordion component is een klikbare (verticaal gestapelde) lijst van koppen om bijbehorende inhoud te verbergen of te vertonen. Met een accordion toon je inhoud op een progressieve manier aan gebruikers.
+Het accordion component is een klikbare (verticaal gestapelde) lijst van koppen om bijbehorende inhoud te verbergen of te tonen. Met een accordion toon je inhoud op een progressieve manier aan gebruikers.
 
-Een accordion bestaat uit meerdere accordionsecties. Gebruik minimaal 3 en maximaal 10 secties onder elkaar. Je kunt door een accordion navigeren met je toetsenbord:
+## Richtlijnen
+
+- Gebruik een accordion op een volle pagina met zowel essentiële als niet-essentiële inhoud. Door niet-essentiële inhoud te verbergen, komen gebruikers sneller bij de primaire inhoud.
```
Comment (VincentSmedinga) [inline packages/css/src/accordion/README.md:7]:
```suggestion
- Gebruik een accordion op een volle pagina met zowel essentiële als niet-essentiële inhoud. Door niet-essentiële inhoud te verbergen, komen gebruikers sneller bij de essentiële inhoud.
```

File: packages/css/src/accordion/README.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+- Gebruikt een accordion niet om teveel essentiële inhoud op 1 pagina te voorkomen. Herschrijf de pagina compacter, verdeel de inhoud over meerdere pagina's of gebruik het (nog te maken) inhoudsopgave-component in plaats van een accordion voor dat doel. Verberg essentiële informatie niet in een accordion.
```
Comment (VincentSmedinga) [inline packages/css/src/accordion/README.md:8]:
```suggestion
- Gebruik een accordion niet om te veel essentiële inhoud op één pagina te voorkomen. Herschrijf de pagina compacter, verdeel de inhoud over meerdere pagina's of gebruik het (nog te maken) inhoudsopgave-component in plaats van een accordion voor dat doel. Verberg essentiële informatie niet in een accordion.
```

**Review by VincentSmedinga — DISMISSED** [review]

File: packages/css/src/accordion/README.md (hunk 2 above)
Comment (alimpens) [reply packages/css/src/accordion/README.md:8]: Haha, the first mistake sounds a little _Den Haags_ :D 

I think we ran into the writing out numbers thing before. Do you propose to deviate from the [Schrijfwijzer](https://www.amsterdam.nl/schrijfwijzer/heldere-taal-basis-onze-huisstijl/tekstonderdelen-heldere-taal/getallen-bedragen-breuken-percentages/)?

Comment (VincentSmedinga) [reply packages/css/src/accordion/README.md:8]: I wouldn’t dare to :) Updated the suggestion.

**Review by VincentSmedinga — APPROVED** [review]

**Pull request merged**